exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -321,8 +327,8 @@ func (prwe *PrwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequ
 	// https://cortexmetrics.io/docs/apis/#remote-api
 	req.Header.Add("Content-Encoding", "snappy")
 	req.Header.Set("Content-Type", "application/x-protobuf")
-	req.Header.Set("X-Prometheus-Remote-Write-Version", "0.1.0")
-	req.Header.Set("User-Agent", "OpenTelemetry-Collector/"+version.Version)
+	req.Header.Set("X-Prometheus-Remote-Write-Version", prwVersion)
```

Comment:
> We need this information as we are using only the User-Agent header information in the back-end AMP service

If this is true, can we remove this line?

Reply:
I do not think that we can remove that line as per the requirement in https://cortexmetrics.io/docs/api/#remote-write.

Reply:
So if it's sent to the backend here we don't need it in the user agent right? It's a bit weird to have the same information in two headers, it's better if the backend can do what it needs based on all the headers.

Reply:
Yes, I understand that. Actually the AMP backend sends the logs to CloudWatch logs and in these logs, only the user-agent field is present. That was the reason I added the `X-Prometheus-Remote-Write-Version` information in the user-agent header. @alolita and @Aneurysm9, could you take a look at this to determine if `X-Prometheus-Remote-Write-Version` information is required in the user-agent header. If not, I can remove it

Reply:
`X-Prometheus-Remote-Write-Version` needs to be a distinct header per the PRW spec.  I'm not sure it needs to be in the user agent header, though.  Particularly if `prwVersion` is a constant, as it is here, we can derive the PRW version from the collector version.  If that ever changes we can re-evaluate in the future.

Reply:
We want it in the user agent header as the other distinct header is not present in the CW Logs from AMP, hence adding it in the user agent header would be way to get this information. Also about deriving the `prwVersion` from the collector version, I do not think they are related and so I don't think it can be derived

Reply:
I agree there will be a 1:1 correspondence between the Collector version and supported PRW protocol version information.